### ISSUE TITLE
How do we define a node ?

### ISSUE THREAD

**Issue by T-FitAndFat** [othalla/puppetdb_exporter#2]
https://github.com/othalla/puppetdb_exporter/blob/ef15e95b653dc02e1c6b59374adc22537fe93700/puppetdb_exporter/puppetdb.py#L28

I have this issue where the catalog environment is set to null on my node, by checking both of those attributes, is it not possible that we might be missing some nodes ? 

I guess my request would be to know if it is also possible to have the total number of nodes no matter the environment given by the exporter ?

**Comment by othalla** [comment]
Yes i think it is possible to have a kind of '*' that match all nodes like in the puppetboard for example.
Is it what you want?

**Comment by T-FitAndFat** [comment]
That sounds about right, so that if we do lose some nodes we are actually aware just by subtracting the sum of all environment nodes from the total
